components/molecule/autosuggest/src/list/index.js

```diff
@@ -0,0 +1,315 @@
+    const nameOnCache = MoleculeSelectList.nameForSessionStorage(element).replace('_featured_', '')
+    window.sessionStorage.removeItem(nameOnCache)
+
+    this.setState({options: this.reworkOptionsWithFeatured(this.optionsReference)})
+  }
+
+  _onFocusRow = (element) => () => {
+    if (!element) {
+      return
+    }
+
+    if ((getCurrentTime() - this.lastTimeArrowIsBinded) <= 500) {
+      return
+    }
+
+    let buffer = this.state.options
+    buffer.forEach((el) => {
+      if (element.getId() === el.getId()) {
+        el.focusRow()
+        this.lastFocusElementPosition = el.getPosition()
+      } else {
+        el.resetFocus()
+      }
+    })
+
+    this.callbackOnFocusRow(element)()
+    this.setState({options: buffer})
+  }
+
+  keyBindings = (event) => {
+    if (this.state.isCollapsed) {
+      return
+    }
+
+    if (event.defaultPrevented) {
+      return
+    }
+
+    switch (event.key) {
+      case 'ArrowDown':
+        this.focusNextItem()
+        event.preventDefault()
+        this.lastTimeArrowIsBinded = getCurrentTime()
+        break
+      case 'ArrowUp':
+        this.focusNextItem(true)
+        event.preventDefault()
+        this.lastTimeArrowIsBinded = getCurrentTime()
+        break
+      case 'Enter':
+        for (let i = 0; i < this.state.options.length; i++) {
+          if (this.state.options[i].isFocus()) {
+            this._onClickRow(this.state.options[i])()
+            this.setState({isCollapsed: true})
+            break
+          }
+        }
+        event.preventDefault()
+        break
+    }
+  }
+
+  focusNextItem (previous = false) {
+    let buffer = this.state.options
+
+    for (let i = 0; i < buffer.length; i++) {
+      if (buffer[i].isFocus()) {
+        buffer[i].resetFocus()
+      }
+    }
+    let lastFocus = this.lastFocusElementPosition
+
+    let nextFocus = (previous) ? lastFocus - 1 : lastFocus + 1
+    let nextScroll = (previous)
+      ? (document.getElementById(BASE_CLASS + '-id').scrollTop - 40)
+      : (document.getElementById(BASE_CLASS + '-id').scrollTop + 40)
+
+    if (nextFocus < 0) {
+      nextFocus = buffer.length - 1
+      nextScroll = (buffer.length - 1) * 40
+    } else if (nextFocus > (buffer.length - 1)) {
+      nextFocus = 0
+      nextScroll = 0
+    }
+
+    buffer[nextFocus].focusRow()
+
+    this.lastTimeArrowIsBinded = 0
+    this.lastFocusElementPosition = nextFocus
+
+    this._onFocusRow(buffer[nextFocus])()
+    this.setState({options: buffer})
+
+    document.getElementById(BASE_CLASS + '-id').scrollTop = nextScroll
+  }
+
+  render () {
+    const className = (this.listSize === SIZES.LARGE)
+      ? (this.state.isCollapsed) ? cx(BASE_CLASS, BASE_CLASS + '-hidden', BASE_CLASS + '-large') : cx(BASE_CLASS, BASE_CLASS + '-large')
+      : (this.state.isCollapsed) ? cx(BASE_CLASS, BASE_CLASS + '-hidden') : cx(BASE_CLASS)
+
+    const atomListWrapper = function () {
+      if (this.iconRemoveItemFeatured && this.iconItemFeatured) {
```

Comment:
This is not needed. This is basci of how React works. What you has to do is send the undefined value in the prop and then manage that in the AtomListRows component instead.

```
return (
          <AtomListRows
            options={this.state.options}
            onFocusCallback={this._onFocusRow}
            onClickCallback={this._onClickRow}
            removeFeatureItem={this._removeFeatureItem}
            iconRemoveItemFeatured={this.iconRemoveItemFeatured}
            iconItemFeatured={this.iconItemFeatured}
          />
```